### PR TITLE
Added implementation of Day1 FizzBuzz problem in Typescript

### PR DESCRIPTION
**Pull request by dasabhishek0790**
Please make sure you have done the following:

- [ ] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
- [ ] Added your Codes
- [ ] Tested your codes
- [ ] Added credits (author and date) at the top of the code
- [ ] Updated README.md
- [ ] Added yourself as a contributor using `all-contributors-cli`

### CODE REVIEW

**Comment by dasabhishek0790** [comment]
PR agains issue https://github.com/CodeToExpress/dailycodebase/issues/19

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
...b>Willian Taiguara Baliza Moura</b></sub>](https://github.com/willianwt)<br />[📖](https://github.com/CodeToExpress/dailycodebase/commits?author=willianwt "Documentation") [💻](https://github.com/CodeToExpress/dailycodebase/commits?author=willianwt "Code") | [<img src="https://avatars3.githubusercontent.com/u/33730790?v=4" width="100px;"/><br /><sub><b>Rajashree Parhi</b></sub>](https://github.com/rajashree23)<br />[📖](https://github.com/CodeToExpress/dailycodebase/commits?author=rajashree23 "Documentation") [💻](https://github.com/CodeToExpress/dailycodebase/commits?author=rajashree23 "Code") | [<img src="https://avatars1.githubusercontent.com/u/40717067?v=4" width="100px;"/><br /><sub><b>Akshit Aggarwal</b></sub>](https://github.com/AkshitAggarwal)<br />[📖](https://github.com/CodeToExpress/dailycodebase/commits?author=AkshitAggarwal "Documentation") [💻](https://github.com/CodeToExpress/dailycodebase/commits?author=AkshitAggarwal "Code") | [<img src="https://avatars1.githubusercontent.com/u/44455626?v=4" width="100px;"/><br /><sub><b>Abhishek</b></sub>](https://github.com/dasabhishek0790)<br />[💻](https://github.com/CodeToExpress/dailycodebase/commits?author=dasabhishek0790 "Code") |
```
Comment (aaditkamat) [inline CONTRIBUTORS.md:13]: The details of @sagunjaiswal  have been removed from the Markdown file. Can you add them back to the file?

**Review by aaditkamat — CHANGES_REQUESTED** [review]
The solution to the problem is correct and presented very elegantly. I just wanted to highlight a small change in the CONTRIBUTORS markdown file, where the details of one of the project contributors has been replaced with your details. Once this small change is made, I can go ahead and approve this PR.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (dasabhishek0790) [reply CONTRIBUTORS.md]: @aaditkamat  : Added, not sure how the contributors file got changed. Please review & merge.

**Review by aaditkamat — APPROVED** [review]
LGTM. I'll go ahead and merge the pull request.

**Pull request merged**